spec/sub_block_spec.rb

```diff
@@ -538,6 +538,40 @@ def initialize
           d.subgroups[2].is_a?(SubItem2).should == true
         end
 
+        it "adding a sub_block_group should override an existing method of that name" do
+          class SB1
+            include Origen::Model
+
+            def blah
+              1
+            end
+          end
+
+          m = SB1.new
+          m.blah.should == 1
+          m.sub_block_group :blah do
+            m.sub_block :my_blah
+          end
+          m.blah.should_not == 1
+        end
+      end
+
+      it "adding a sub_block should not override an existing method of that name" do
```

Comment:
@ginty Should this read 'adding a sub_block should override...', i.e. remove the 'not' ?